test/spec/select.txt

```diff
@@ -2,49 +2,49 @@
 ;;; STDIN_FILE: third_party/testsuite/select.wast
 (;; STDOUT ;;;
 out/test/spec/select.wast:290: assert_invalid passed:
-  error: type mismatch in select, expected [i32, any, any] but got [i32]
+  error: type mismatch in select, expected [any, any, i32] but got [i32]
```

Comment:
is this just a copy of the upstream spec test? Is the reference interpreter output wrong too?

Reply:
This output is wabt-specific I believe, the spec tests don't tests these parts of the error.

Reply:
We verify the full error message here, but the spec tests will as long as "type mismatch" is part of the message.